app/models/generic.rb

```diff
@@ -2,19 +2,17 @@
 #  `rails generate hyrax:work Generic`
 class Generic < ActiveFedora::Base
   include ::Hyrax::WorkBehavior
-  include ::OregonDigital::TriplePoweredProperties::WorkBehavior
+  #include ::OregonDigital::TriplePoweredProperties::WorkBehavior
+  include OregonDigital::GenericMetadata
+  include ::Hyrax::BasicMetadata
 
   self.indexer = GenericIndexer
+
   # Change this to restrict which works can be added as a child.
   # self.valid_child_concerns = []
   validates :title, presence: { message: 'Your work must have a title.' }
 
-  # oEmbed property
-  property :oembed_url, predicate: ::RDF::Vocab::DC.abstract do |index|
-    index.as :stored_searchable
+  def set_defaults
+    super
```

Comment:
@straleyb I tried removing `super` locally since I was getting `super: no superclass method `set_defaults' for #` on my local. Are you getting this error too? I'm thinking maybe we need something like default in ScholarsArchive https://github.com/osulp/Scholars-Archive/blob/master/app/models/default.rb#L22-L23?

Reply:
You know, I didnt even notice that, but yeah, I thought this method was coming from somewhere, so thats why I called super. Thanks for the tip!